README.md

```diff
@@ -116,9 +98,53 @@ conda create -n py310 python=3.10 anaconda
 ```
 In this example, the name of the environemnt is the same as the python version, but it have any name (depending on your needs).
```

Comment:
I know that this was before, but could you correct it also please? Thank you

Reply:
https://github.com/dh-epfl-students/student-project-howtos/pull/4/commits/b69c2fb02a144e48169ceffcab96e2bd56ba249d